packages/gsmevil2/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=gsmevil2
+pkgver=12.eb96b4f
+pkgrel=1
+pkgdesc='Python web-based tool which use for capturing imsi numbers and sms.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-radio' 'blackarch-sniffer')
+url='https://github.com/ninjhacks/gsmevil2'
+license=('custom:unknown')
+depends=('gr-gsm' 'kalibrate-rtl' 'python-flask' 'python-flask-socketio' 'python-pyshark')
```

Comment:
some dependencies are missing

![image](https://github.com/user-attachments/assets/20bc54ab-41c7-438d-9158-3983547b2bbf)

please test installing and runing the package in minimal environment

either with ba-dev or https://wiki.archlinux.org/title/DeveloperWiki:Building_in_a_clean_chroot

Reply:
Sorry if I didn't use ba-dev in my last PRs. Now I retrieved it and fixed PKGBUILD according ba-dev output. Let me know if it's ok. Thanks